ray-operator/controllers/ray/utils/util.go

```diff
@@ -94,6 +100,16 @@ func GenerateServiceName(clusterName string) string {
 	return fmt.Sprintf("%s-%s-%s", clusterName, rayiov1alpha1.HeadNode, "svc")
 }
 
+// GenerateIngressName generates an ingress name from cluster name
+func GenerateIngressName(clusterName string) string {
+	return fmt.Sprintf("%s-%s-%s", clusterName, rayiov1alpha1.HeadNode, "ingress")
+}
+
+// GenerateRayClusterName generates a ray cluster name from ray service name
+func GenerateRayClusterName(serviceName string) string {
+	return fmt.Sprintf("%s%s%s", serviceName, RayClusterSuffix, rand.String(5))
```

Comment:
I'm slightly concerned about naming collisions due to the truncation that happens all over the place
https://github.com/ray-project/kuberay/blob/37d39954faad220f96ac93efae0e3f32adedc142/ray-operator/controllers/ray/utils/util.go#L24,
but maybe this is unlikely to cause issues.

Reply:
yes, this is a point we should be careful.